templates/common/control_header.html

```diff
@@ -13,10 +13,10 @@
 		<title>(Puerto Plata Express - {% block title %}{% endblock %})</title>
```

Comment:
Why two headers files with almost the same structure, templates inheritance is for avoid this.